src/ui/src/components/blueprints/abstract/BlueprintsNode.vue

```diff
@@ -527,6 +527,13 @@ watch(isEngaged, () => {
 	height: 24px;
 }
 
+.BlueprintsNode__main__title .skeleton {
+	max-width: 24px;
+	max-height: 24px;
+	aspect-ratio: 1 / 1;
+	border-radius: 4px;
+}
```

Comment:
This should not work. The CSS here is scoped to the component, so `.skeleton` is not accessible.

You could use `:deep` to traverse the scope.

```suggestion
.BlueprintsNode__main__title :deep(.skeleton) {
	max-width: 24px;
	max-height: 24px;
	aspect-ratio: 1 / 1;
	border-radius: 4px;
}
```

But I think the better solution could be to add a prop for `SharedImgWithFallback`. So maybe something like

```vue
<SharedImgWithFallback.vue :url loader-max-width-px="24" loader-max-height-px="24" />
```

Reply:
Yes I was thinking about this too, let me check this.
```<SharedImgWithFallback.vue :url loader-max-width-px="24" loader-max-height-px="24" />```